docs/releasenotes/10.2.0.rst

```diff
@@ -0,0 +1,56 @@
+10.2.0
+------
+
+Backwards Incompatible Changes
+==============================
+
+TODO
+^^^^
+
+TODO
+
+Deprecations
+============
+
+TODO
+^^^^
+
+TODO
+
+API Changes
+===========
+
+TODO
+^^^^
+
+TODO
+
+API Additions
+=============
+
+Added DdsImagePlugin enums
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:py:class:`PIL.DdsImagePlugin.DDSD`, :py:class:`PIL.DdsImagePlugin.DDSCAPS`,
```

Comment:
Let's add the tilde to all these, so it only shows the enum name not its full path. 

https://pillow--6486.org.readthedocs.build/en/6486/releasenotes/10.2.0.html#added-ddsimageplugin-enums


```suggestion
:py:class:`~PIL.DdsImagePlugin.DDSD`, :py:class:`~PIL.DdsImagePlugin.DDSCAPS`,
```